config/routes.rb

```diff
@@ -223,12 +223,10 @@
   resources :partnerships, only: %i[index create show], param: :option
   resources :display_ad_events, only: [:create]
   resources :badges, only: [:index]
-  resource :pro_membership, path: :pro, only: %i[show create update]
   resources :user_blocks, param: :blocked_id, only: %i[show create destroy]
   resources :podcasts, only: %i[new create]
   resources :article_approvals, only: %i[create]
   resources :video_chats, only: %i[show]
-  resolve("ProMembership") { [:pro_membership] } # see https://guides.rubyonrails.org/routing.html#using-resolve
```

Comment:
me: "what is this"
also me: "guess i don't need to know now!!"

Reply:
(i still clicked the guides link, lol)

Reply:
I think I originally wrote that line. I have no idea what it does, I didn't click the link, I want to forget how I got there :D